### ISSUE TITLE
issue merge styles from media="print" to global styles 

### ISSUE THREAD

**Issue by huukhai87** [painty/CSS-Used-ChromeExt#10]
1. The url of page inspected.

`https://www.everydiam.com/diamond-jewelry/sku30191.html`

2. The element inspected.

`div.wrapper-bottom`

3. Expected result.

```css
.wrapper-bottom {
    background: transparent url("https://www.everydiam.com/skin/frontend/default/forest_fancyshop/images/footer_bg.gif") repeat-x scroll center bottom;
}
```

4. Actual result.

```css
.wrapper-bottom {
    background: transparent url("https://www.everydiam.com/skin/frontend/default/forest_fancyshop/images/footer_bg.gif") repeat-x scroll center bottom;
}
// more style from media="print"
* {
    background: none!important; 
    text-align: left!important;
}
* {
    margin: 0;
    padding: 0;
}
```

note: the style
```css
* {
    background: none!important; 
    text-align: left!important;
}
* {
    margin: 0;
    padding: 0;
}
```
from link media="print" href="https://www.everydiam.com/media/css_secure/6a4b386e11eb511b87056ed631be9245.css" type="text/css" rel="stylesheet",

 pls don't merge styles code from media print and just keep styles from media ="all"

**Comment by painty** [comment]
I'll preserve the media property of link / styleTag , in the next release.

**Comment by painty** [comment]
Fixed in 2.2.7, goto chrome://extensions/?id=cdopjfddjlonogibjahpnmjpoangjfff and update.
